tools/scripts/pub.mjs

```diff
@@ -97,16 +98,23 @@ await question('Are you sure you want to publish to? (y/n)', {
       const pkg = await readJsonFile(`${dir}/package.json`);
 
       // also read the individual package.json and update the version
-      try{
+      try {
         const pkg2 = await readJsonFile(
           `${dir.replace('dist/', '')}/package.json`
         );
         pkg2.version = lernaVersion;
 
         // write the package.json file
         await writeJsonFile(`${dir.replace('dist/', '')}/package.json`, pkg2);
-      }catch(e){
-        // swallow
+      } catch (e) {
+
+        const path = `${dir.replace('dist/', '')}/package.json`;
+
+        // swallow error if it's not a vanilla package
+        if(!path.includes('vanilla')){
```

Comment:
I will add a guard that if the package includes the `vanilla` keyword it will swallow, else log the error, i think that would be better

Reply:
perfect